docs/book/tutorials/deploying_cpi_and_csi_with_multi_dc_vc_aka_zones.md

```diff
@@ -67,35 +67,42 @@ Steps that will be covered in order to setup zones for the vSphere CPI, vSphere
 
-The zones implementation depends on 2 sets of vSphere tags to be used on objects, such as datacenters or clusters. The first is a `region` tag and the second is a `zone` tag. vSphere tags are very simply put key/value pairs that can be assigned to objects and instead of using fixed keys to denote a `region` or a `zone`, we give the end-user the ability to come up with their own keys for a `region` and `zone` in the form of vSphere Tag Catagory. It just allows for a level of indirection in case you already have regions and zones setup in your configuration. Once a key/label or vSphere Tag Category is selected for each, create a `[Labels]` section in the `vsphere.conf` then assign tag names for both `region` and `zone`.
+The zones implementation depends on 2 sets of vSphere tags to be used on objects, such as datacenters or clusters. The first is a `region` tag and the second is a `zone` tag. vSphere tags are very simply put key/value pairs that can be assigned to objects and instead of using fixed keys to denote a `region` or a `zone`, we give the end-user the ability to come up with their own keys for a `region` and `zone` in the form of vSphere Tag Catagory. It just allows for a level of indirection in case you already have regions and zones setup in your configuration. Once a key/label or vSphere Tag Category is selected for each, create a `labels:` section in the `vsphere.conf` then assign tag names for both `region` and `zone`.
 
 In the example `vsphere.conf` below, `k8s-region` and `k8s-zone` was selected:
 
 ```bash
-[Global]
-# properties in this section will be used for all specified vCenters unless overridden in VirtualCenter section.
-
-user = "vCenter username for cloud provider"
-password = "password"
-port = "443" #Optional
-insecure-flag = "1" #set to 1 if the vCenter uses a self-signed cert
-datacenters = "list of datacenters where Kubernetes node VMs are present"
-
-[VirtualCenter "1.2.3.4"]
-# Override specific properties for this Virtual Center.
-        user = "vCenter username for cloud provider"
-        password = "password"
-        # port, insecure-flag, datacenters will be used from Global section.
-
-[VirtualCenter "10.0.0.1"]
-# Override specific properties for this Virtual Center.
-        port = "448"
-        insecure-flag = "0"
-        # user, password, datacenters will be used from Global section.
-
-[Labels]
-region = k8s-region
-zone = k8s-zone
+# Global properties in this section will be used for all specified vCenters unless overriden in VirtualCenter section.
+global:
+  user: YourVCenterUser
+  password: YourVCenterPass
+  port: "443"
+  # set insecure-flag to true if the vCenter uses a self-signed cert
+  insecure-flag: true
+  # settings for using k8s secret
+  secret-name: cpi-secret
+  secret-namespace: kube-system
+
+# VirtualCenter section
+virtualcenter:
+  tenant1:
```

Comment:
In the future we should support a list for this, something like:
```yaml
virutalcenter:
  - name: tenant1
    user: ...
    password: ...
  - name: tenant2
    user: ..
```

Reply:
we should really change `virtualcenter` to `vcenter` as well, the Virtual Center name has been deprecated 10 years ago 😄

Reply:
changed `virtualcenter:` to `vcenter:` in code and in docs